dev/release/verify-release-candidate.sh

```diff
@@ -531,16 +530,34 @@ test_integration() {
               $INTEGRATION_TEST_ARGS
 }
 
-clone_testing_repositories() {
-  # Clone testing repositories if not cloned already
-  if [ ! -d "arrow-testing" ]; then
-    git clone https://github.com/apache/arrow-testing.git
+get_source_distribution() {
+  ARROW_DIR="$(dirname $(dirname ${SOURCE_DIR}))"
```

Comment:
`ARROW_DIR` should be the extracted source archive directory.

Reply:
Ok. Need to check all works correctly.

Reply:
Fixed.